jormungandr/src/main.rs

```diff
@@ -44,7 +44,9 @@ extern crate serde_json;
 extern crate serde_yaml;
 #[macro_use(b, o, record_static, debug, info, warn, error, crit)]
 extern crate slog;
+extern crate hostname;
```

Comment:
now that you use uuid. do you still need this?

Reply:
fixed.